tests/integration_tests/performance/configs/test_network_tcp_throughput_config_4.14.json

```diff
@@ -2507,128 +2507,128 @@
                                         "1vcpu_1024mb.json": {
                                             "total": {
                                                 "tcp-p1024K-ws16K-g2h": {
-                                                    "target": 4181,
-                                                    "delta_percentage": 6
+                                                    "target": 3897,
+                                                    "delta_percentage": 29
                                                 },
                                                 "tcp-p1024K-ws256K-g2h": {
-                                                    "target": 31026,
-                                                    "delta_percentage": 29
+                                                    "target": 24078,
+                                                    "delta_percentage": 90
                                                 },
                                                 "tcp-p1024K-wsDEFAULT-g2h": {
-                                                    "target": 43625,
-                                                    "delta_percentage": 36
+                                                    "target": 32568,
+                                                    "delta_percentage": 135
                                                 },
                                                 "tcp-pDEFAULT-ws16K-g2h": {
-                                                    "target": 4179,
-                                                    "delta_percentage": 7
+                                                    "target": 3862,
+                                                    "delta_percentage": 29
                                                 },
                                                 "tcp-pDEFAULT-ws256K-g2h": {
-                                                    "target": 29993,
-                                                    "delta_percentage": 27
+                                                    "target": 19986,
+                                                    "delta_percentage": 142
```

Comment:
This delta is kind of worrying...I see all  deltas are quite big;; did you gather these with a big number of runs?

Reply:
I ran again with 30 runs/batch and updated the PR, but the deltas are still pretty big. In the meantime I will be looking to see if there is anything that we could do to improve stability.